### PR TITLE
Alex/improvements

### PR DESCRIPTION
**Pull request by asoong**
- Add tests for rebalanceSetToken minimum proposal period and rebalance interval back in
- Add tests for top level setProtocol.js methods
- Create helper method for deploying base contracts

### CODE REVIEW

File: test/helpers/index.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export {
   addAuthorizationAsync,
   getTokenBalances,
   tokenDeployedOnSnapshot,
+  deployBaseContracts
```
Comment (justinkchen) [inline test/helpers/index.ts]: nit trailing comma

Comment (asoong) [reply test/helpers/index.ts]: +1
